pkg/rbd/rbd_journal.go

```diff
@@ -183,9 +197,19 @@ func checkVolExists(ctx context.Context, rbdVol *rbdVolume, cr *util.Credentials
 				rbdVol.RbdImageName, rbdVol.RequestName)
 			return false, err
 		}
+
 		return false, err
 	}
-
+	if clone {
+		ok, flatErr := checkIsFlattenInProgress(ctx, rbdVol, cr)
```

Comment:
Same comment here -- if we are restarting a create volume request, we just re-use the existing flatten logic.